test/system/500-networking.bats

```diff
@@ -163,6 +163,7 @@ load helpers
 
 # "network create" now works rootless, with the help of a special container
 @test "podman network create" {
+    skip_if_netavark "Broken with netavark: https://github.com/containers/podman/issues/13072"
```

Comment:
Can remove this skip and try again I think we fixed many aardvark bugs and this works locally for me.

Reply:
Happy to, but I think I need to wait for https://github.com/containers/podman/pull/12859 to merge, since (I'm assuming) it also fixes the `Netavark integaration` failures happening here.

Reply:
okay, I removed the commit that added this, lets see how it goes...